recipes-sota/aktualizr/aktualizr-callback.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Aktualizr configuration snippet to enable Foundries.IO callback function"
+SECTION = "base"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7f619bab123dad"
+
+inherit allarch
+
+SRC_URI = "\
+    file://90-handle-callback.toml \
+    file://callback-handler \
+"
+
+PV = "1.0"
```

Comment:
No need for PV directly here, you can just rename the recipe instead (aktualizr-callback_1.0.bb).